influxdb3_server/src/system_tables/mod.rs

```diff
@@ -123,6 +125,10 @@ impl AllSystemSchemaTablesProvider {
             ))),
         );
         tables.insert(PARQUET_FILES_TABLE_NAME, parquet_files);
+        let logs_table = Arc::new(SystemTableProvider::new(Arc::new(
+            ProcessingEngineLogsTable::new(sys_events_store),
```

Comment:
How many log events will this store? Is this configurable?

Reply:
The default is `MAX_CAPACITY`, which is 10_000. This isn't currently configurable. It is used as the size of the ring buffer for all entries in the `SysEventStore`.

Reply:
Hmmm, it would be better to be able to split this out on a per table basis. I'm guessing the plugin log is going to be very chatty relative to others. Add in a follow on PR?

Reply:
Yeah, lets get that in a follow on PR.